### ISSUE TITLE
yt-healthcheck dumps warnings when --dump-detail and LongQuery includes `%` character

### ISSUE THREAD

**Issue by yoku0825** [yoku0825/ytkit#31]
```
Missing argument in printf at /usr/local/share/perl5/Ytkit.pm line 153.
Invalid conversion in printf: "%Y" at /usr/local/share/perl5/Ytkit.pm line 153.
Missing argument in printf at /usr/local/share/perl5/Ytkit.pm line 153.
..
WARNING on host1: Detected long query(36 sec) by u1@xxx:xxx: "SELECT  COUNT(*)  FROM  .. WHERE col1 >  DATE_FORMAT(NOW(),'%Y%m0%H0')  and ..
```

**Comment by yoku0825** [comment]
```
$ perl -Ilib -MYtkit -E 'my $ytkit = {}; bless $ytkit => "Ytkit"; $ytkit->infof("%s:%d", "SELECT * FROM %Y", 3306)'
Missing argument in printf at lib/Ytkit.pm line 153.
Invalid conversion in printf: "%Y" at lib/Ytkit.pm line 153.
SELECT * FROM %Y:3306
```

**Comment by yoku0825** [comment]
Fixed by 4f715a1
